src/app/views/organization/organization.html

```diff
@@ -34,17 +34,18 @@ <h1 translate>mnoe_admin_panel.dashboard.organization.organization_info</h1>
                   {{vm.organization.external_id}}
                 </div>
               </div>
-              <div class="row">
+              <!-- For KPMG -->
```

Comment:
I don't think we should mention any clients explicitly since this is an open source project. I'd remove this comment and maybe just explain what the commented out section does, in the event the user wants to make use of it.